plasma_framework/test/src/vaults/Erc20Vault.test.js

```diff
@@ -176,6 +176,33 @@ contract('Erc20Vault', (accounts) => {
             expect(postBalance).to.be.bignumber.equal(expectedPostBalance);
         });
 
+        it('should fail when called from under quarantine vault', async () => {
```

Comment:
It should be under quarantine **Exit Game** instead of vault here.

I would break this into two tests to test one thing at once. One for this and another testing withdraw can success when quarantined period passes.

If you really want to group things together, I think we can leverage `describe` and do sth like below. 
```
describe('Given non quarantined exit game', () => {
   it ('should fail when called by the exit game contract', ...);

  describe('And then the quarantined period passes', () => {
        beforeEach(() => { pass the period.... })

       it('should run the withdraw successfully', ...);
  })
})
```

Reply:
Really nice idea - thanks!